build-system/global-configs/client-side-experiments-config.json

```diff
@@ -1,5 +1,9 @@
 {
   "experiments": [
+    {
+      "name": "flexible-bitrate",
+      "percentage": 0.1,
+    },
     {
```

Comment:
If you don't mind, please remove test-amp-exp - we verified the test and are ready to delete it